OWNERS

```diff
@@ -31,3 +31,4 @@ approvers:
   - jschintag
   - c4rt0
   - cverna
+  - Adam0Brien
```

Comment:
This is now missing an empty line at the end of the file.

Reply:
added the empty line